tests/dragonfly/connection_test.py

```diff
@@ -610,18 +613,29 @@ async def test_keyspace_events_config_set(async_client: aioredis.Redis):
 async def test_reply_count(async_client: aioredis.Redis):
     """Make sure reply aggregations reduce reply counts for common cases"""
 
-    async def get_reply_count():
-        return (await async_client.info("STATS"))["reply_count"]
+    port = int(async_client.get_connection_kwargs().get("port"))
+    METRICS_URL = f"http://localhost:{port}/metrics"
+    logging.getLogger("urllib3").setLevel(logging.WARNING)
+
+    def get_reply_count():
+        response = requests.get(METRICS_URL)
+        response.raise_for_status()
+        families = text_string_to_metric_families(response.text)
+        for metric in families:
+            if metric.name == "dragonfly_reply":
+                assert metric.type == "counter"
+                return int(metric.samples[0].value)
+        return None
```

Comment:
There is `DfInstance.metrics` that is even async, you just need to get the `df_server` fixture